### DISCUSSION TITLE
Upgrading from 2.3.2 to 2.3.3 results in persistent FileTaskHandler warning

### DISCUSSION THREAD

**Discussion by rhaussmann** [apache/airflow#25381] [Q&A]
I'm trying to upgrade from airflow 2.3.2 to 2.2.3 in a docker environment. When I upgraded to 2.3.3, I started getting a persistent warning in my webserver logs. It appears as often as every other line, making reading the log files difficult. Otherwise, it doesn't appear to be impacting anything. 

It seems to be originating in the method `providers.amazon.aws.log.s3_task_handler.S3TaskHandler ` When I set `remote_logging` to `False` in the `airflow.cfg` file, the warnings stop. 

The warning:

`/usr/local/lib/python3.8/site-packages/airflow/utils/log/file_task_handler.py:52 DeprecationWarning: Passing filename_template to FileTaskHandler is deprecated and has no effect`

I tried putting the `airflow_local_settings.py` file in the config directory with all the references to `filename_template` commented out. This didn't make any difference.

The only other place I've seen this is in the logs in [a discussion about another problem](https://github.com/apache/airflow/discussions/25109).

 Is there a solution to this issue?

**Comment by potiuk** [comment]
This has been added here - https://github.com/apache/airflow/commit/c23826915dcdca4f22b52b74633336cb2f4a1eca looks like you need to remove `filename_template` from the configuration. 

Did you do it ?
